### PR TITLE
Fixed bug for missing organisational_testresult

### PR DESCRIPTION
**Pull request by shilpigoeldev**
## What?
Some of the Bristol records were missing ORGANISATIONCODE_TESTRESULT .  More details on Jira #2056

## Why?
Genotype duplication logic was wrong and rather duplicating a new genotype, it was being created from raw record and hence was loosing on the properties already set.

## How?
Logic has been rightly applied for duplicating genotype and also code has been restructured.

## Test? 

```select * from molecular_data where organisationcode_testresult is null;  ``` gives 0 rows now.

### CODE REVIEW

File: lib/import/brca/providers/bristol/bristol_handler.rb [hunk 1]
```diff
@@ -8,66 +6,57 @@ module Bristol
...
-            duplicated_genotype.add_test_scope(:full_screen)
-            duplicated_genotype.add_passthrough_fields(record.mapped_fields,
-                                                       record.raw_fields,
-                                                       PASS_THROUGH_FIELDS)
             genotypes.append(duplicated_genotype)
           end
 
+          def process_positive_record(genotype, positive_gene, record, genotypes)
+            process_cdna_change(genotype, record)
+            genotype.add_gene_location(record.mapped_fields['codingdnasequencechange'])
```
Comment (lauramccluskey1) [inline lib/import/brca/providers/bristol/bristol_handler.rb:35]: Is this overwriting the add_gene_location in process_cdna_change?

Comment (shilpigoeldev) [reply lib/import/brca/providers/bristol/bristol_handler.rb]: Right @lauramccluskey1 , unnecessary code.. removed now in latest commit, thank you !

**Comment by lauramccluskey1** [comment]
Code changes look good to me- have you checked the counts are the same before and after?

**Comment by shilpigoeldev** [comment]
> Code changes look good to me- have you checked the counts are the same before and after?

Yes , that I have checked ✅

**Review by lauramccluskey1 — APPROVED** [review]

**Pull request merged**